.env.example

```
@@ -1,13 +1,50 @@
+JWT_SECRET=0exyGqsGgbtyx36bStLrM38/XzzGSgcrv2RCC0b6ylk=
+JWT_ACCESS_TOKEN_EXPIRATION=1800000
+JWT_REFRESH_TOKEN_EXPIRATION=604800000
+
+
+# Config-Server
+CONFIG_SERVER=localhost:13100
+
+
+# JPA 설정(배포 환경에서 초기 구동 후 수정 필요)
+# 배포 시 ddl-auto는 validate, show-sql은 false로 변경
+SPRING_JPA_HIBERNATE_DDL-AUTO=update
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The SPRING_JPA_HIBERNATE_DDL-AUTO key has incorrect delimiter

(IncorrectDelimiter)